mapping/requirements_officejs.json

```diff
@@ -5034,6 +5072,17 @@
 							}],
 							"availability": "GA"
 						},
+						{
+							"name": "WordApi",
+							"apiVersion": "1.8",
+							"supportedProductVersions": [{
+								"from": {
+									"build": "16.0.17628.20110",
+									"version": "2311"
```

Comment:
Is this version correct? It's the same as the version used for the prior, WordApi 1.7 build (16.0.17029.20068)

Reply:
thanks, correct the version.